client/signup/steps/about/index.jsx

```diff
@@ -254,23 +254,25 @@ class AboutStep extends Component {
 		const siteTitleInput = formState.getFieldValue( this.state.form, 'siteTitle' );
 		const siteGoalsInput = formState.getFieldValue( this.state.form, 'siteGoals' );
 		const siteGoalsArray = siteGoalsInput.split( ',' );
+		const siteGoalsGroup = siteGoalsArray.sort().join();
```

Comment:
this ensures that the order is always the same.

Reply:
is there any reason for two variables?

```js
const siteGoalsArray = siteGoalsInput.split( ',' ).sort().join();
```